cmake/PythonModules.cmake

```diff
@@ -86,7 +86,7 @@ function(py_add_module NAME)
     )
 
 endfunction()
-set(PYTHON_SEARCH_VERSIONS 3.5 3.6 3.7 3.8 3.9 3.10)
+set(PYTHON_SEARCH_VERSIONS 3.5 3.6 3.7 3.8 3.9 3.10 3.12)
```

Comment:
Do we still want to add python 3.11?

Reply:
Numpy v1.24+ has python3.11 support. 
Onnx==1.13+ has support for python3.11 support. 
Also we would need to  figure out how to test it in CI , which would be a bit difficult. 
We can add it later if necessary.

Reply:
Agreed this should be a separate ticket/issue as well should we need the coverage

Reply:
added python3.11. So that migraphx would build bindings for it.